packages/server/.env

```diff
@@ -34,7 +34,7 @@ STATIC_ROOT=../../ui/build
 
 # In production leave this empty to make the UI connect to the nginx reverse proxy.
 # In testing we can point to grpcwebproxy (started in `hack/docker`) which is configured to allow CORS.
-UI_GRPC_URL=https://localhost:8443
+UI_GRPC_URL=https://clarity.casperlabs.io
```

Comment:
I think it should still be pointing at localhost.

Reply:
I could be wrong but I think @Abner suggested committing this change to the repo when I asked him previously?

Reply:
Yeah, it doesn't matter, production and hack/docker use the env variables defined docker-compose.yml.

Reply:
So let's keep the localhost here.